Documentation/ReadMe.txt

```diff
@@ -5,6 +5,7 @@ A brief list of the relevant files is included here:
 2) CameraVel.py = Secondary Camera script which handles camera feed and overlays speed in km/hr onto the display. Uses a reed switch input.
 3) convert.sh = Shell script for handling conversion of video files at the end of a run. Also manages SD card storage of video files so SD card does not get too full.
 4) run.py = Master script which indicates status of system to external observers and runs other files based on input from a user.
+5) orchestrator_test.py = Test script for orchestrator.py. To run enter 'pytest' in the terminal to run the tests
```

Comment:
Wrong readme - I believe this file is from the V1.5 camera system but I'm not sure. It needs to be removed at some point. The one that this should go in is in the root folder, titled `README.md`. Just make a new section called "Tests" or something like that.